Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/Controller.purs

```diff
@@ -151,6 +205,22 @@ eval (APIFailureActionController (ErrorModal.PrimaryButtonActionController Prima
 
 eval (NoRidesActionController (ErrorModal.PrimaryButtonActionController PrimaryButton.OnClick)) state = exit GoHome
 
+eval (EmailEditTextAC (PrimaryEditText.TextChanged id a)) state = continue state{data {email = trim(a)},props{btnActive = if ((length (trim(a)) > 0 ) && (length state.data.description > 2) && (validateEmail a)) then true else false}}
+
+eval (DescriptionEditTextAC (PrimaryEditText.TextChanged id a)) state = continue state{data {description = a},props{btnActive = if ((length state.data.email > 0) && (length a > 2) && (validateEmail state.data.email)) then true else false}}
```

Comment:
here also try using the condition itself, instead of separately writing true and false.